src/main/content/about.html

```diff
@@ -36,7 +36,7 @@ <h2 id="intro_subtitle_2">Why open source Liberty?</h2>
                     Today, we're striving to make Liberty even more user-friendly, and the right fit for your application in any environment you choose to run.
                     Looking toward the future, we believe the combination of Java, open standards and open source is the best way to enable developers to
                     create amazing applications. So, last year we got together with Red Hat, Payara, Tomitribe, and others to accelerate Java for creating
-                    Microservices, and we're taking the next step by creating open source from our MicroProfile<sup>TM</sup> runtime and Java EE. 
+                    Microservices, and we're taking the next step by creating open source from our MicroProfile&reg; runtime and Java EE. 
```

Comment:
We don't need this reg, only for first usage. Also the &reg; is for Eclipse MicroProfile so this usage was always suspect.